openedx/adg/lms/webinars/handlers.py

```diff
@@ -0,0 +1,19 @@
+"""
+handler methods for webinars
+"""
+
```

Comment:
Remove this empty line as decided internally to have an empty line only for class level docstrings.

Reply:
And it would be better to write Handler functions instead of methods. Methods usually belong to objects/classes, and these standalone helpers or handlers are called functions.